qiskit/algorithms/time_evolvers/time_evolution_result.py

```diff
@@ -1,6 +1,6 @@
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2021, 2022.
+# (C) Copyright IBM 2021, 2022, 2023.
```

Comment:
```suggestion
# (C) Copyright IBM 2021, 2023.
```
The format is `first year` or `first year, last year` if the last year if any change is different from the first. (Any intermediate years of change are not listed)